include/OsmAndCore/TileSqliteDatabase.h

```diff
@@ -97,6 +97,7 @@ namespace OsmAnd
         bool isTileTimeSupported() const;
         bool hasTimeColumn() const;
         bool enableTileTimeSupport(bool force = false);
+        void enableTileTimeSupportIfNeeded();
```

Comment:
Drop it. Use exists methods like isTileTimeSupported, hasTimeColumn, enableTileTimeSupport